catalog.css

```diff
@@ -0,0 +1,309 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  font-family: "RobotoRegular", Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #283136;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+header {
+  background-color: #eeeeee;
+}
+.header-logo:hover,
+.header-logo:focus {
+  opacity: 0.8;
+}
+.header-logo:active {
+  opacity: 0.3;
+}
+
+.header-navigation ul {
+  list-style: none;
+}
+.header-navigation a,
+.cart a {
+  font-family: "RobotoMedium", Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 30px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: black;
+}
+header h1 {
+  font-family: "RobotoMedium", Helvetica, sans-serif;
+  font-size: 55px;
+  line-height: 55px;
+
+  color: black;
+}
+
+.header-navigation a:hover,
+.header-navigation a:focus,
+.cart a:hover,
+.cart a:focus {
+  color: #fb565a;
+}
+.header-navigation a:active,
+.cart a:active {
+  color: #a6a6a6;
+}
+
+.filter h2 {
+  font-family: "RobotoBold", Helvetica, sans-serif;
+  font-size: 18px;
+  line-height: 30px;
+
+  text-transform: uppercase;
+
+  color: black;
+}
+.price-in-range label {

```

Comment:
common, it's a dublicate

Reply:
```
.price-in-range label {
 line-height: 22px;
}
.price-in-range output {
  line-height: 22px;
}
```
---->> 
.price-in-range label,
.price-in-range output {
 line-height: 22px;
}